docs/end-user/component-delivery.md

```diff
@@ -2,4 +2,210 @@
+
+## KubeVela's Component
+
+We use [KubeVela CLI][1] to check out all the available Components:
+
+```shell
+$ vela components
+NAME        NAMESPACE  	WORKLOAD                             	DESCRIPTION                                                            
+alibaba-ack	vela-system	configurations.terraform.core.oam.dev	Terraform configuration for Alibaba Cloud ACK cluster       
+alibaba-oss	vela-system	configurations.terraform.core.oam.dev	Terraform configuration for Alibaba Cloud OSS object        
+alibaba-rds	vela-system	configurations.terraform.core.oam.dev	Terraform configuration for Alibaba Cloud RDS object        
+helm       	vela-system	autodetects.core.oam.dev             	helm release is a group of K8s resources from either git    
+           	           	                                     	repository or helm repo                                     
+kustomize  	vela-system	autodetects.core.oam.dev             	kustomize can fetching, building, updating and applying     
+           	           	                                     	Kustomize manifests from git repo.                          
+raw        	vela-system	autodetects.core.oam.dev             	raw allow users to specify raw K8s object in properties     
+task       	vela-system	jobs.batch                           	Describes jobs that run code or a script to completion.     
+webservice 	vela-system	deployments.apps                     	Describes long-running, scalable, containerized services    
+           	           	                                     	that have a stable network endpoint to receive external     
+           	           	                                     	network traffic from customers.                             
+worker     	vela-system	deployments.apps                     	Describes long-running, scalable, containerized services    
+           	           	                                     	that running at backend. They do NOT have network endpoint  
+           	           	                                     	to receive external network traffic.    
+```
+
+Let's take several typical component types as example to introduce the usage of KubeVela component delivery. If you want to directly view how cloud resources are used, please read [Integrate Cloud Service][2].
+
+
+## Using service-based components (webservice)
```

Comment:
We should not expose the concept of webservice type to users...

Reply:
you mean we don't have to introduce the using case of webservice, Let them try by themself?